### ISSUE TITLE
How to fix nodetree when it breaks ?

### ISSUE THREAD

**Issue by gordielachance** [JacquesLucke/animation_nodes#902]
Hi, my nodetree has a red border and the execution time (ms) is not updated anymore.
I guess this is because of an error.

How can I know which node fails so I can try to fix it, and is it even possible ? I have the feeling that when it breaks, even if I delete almost everythings, it stays broken.

**Comment by JacquesLucke** [comment]
There is more information in the left side bad of the node editor and in the terminal.
Furthermore please let me know which OS and version of AN you use. Also it would be nice to have a screenshot and the .blend file if possible.

**Comment by gordielachance** [comment]
> There is more information in the left side bad of the node editor and in the terminal.

Okay

> Furthermore please let me know which OS and version of AN you use.

Windows, 2.0.4

BTW is it mac to pc & pc to mac compatible ?  Because I have a mac too and I usually work on both OS.

In this case, I just saved my file while AN was working.
Then I reopened Blender (had to re-enable the addon since it was now disabled) and AN is frozen.

screenshot : https://imgur.com/OAUZ9h0
file : https://www.dropbox.com/s/v3dzh8gglnibl39/gridTest6-PC.blend?dl=0

Thanks for ur help !
